code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -744,12 +744,60 @@
 		var/whack_speed = (prob(60) ? 1 : 4)
 		target.throw_at(throw_target, rand(1, 2), whack_speed, user)
 
+
+/obj/item/ego_weapon/moonlight
+	name = "moonlight"
+	desc = "The serpentine ornament is loyal to the original owner’s taste. The snake’s open mouth represents the endless yearning for music."
+	special = "Use this weapon in hand to heal the sanity of those around you."
+	icon_state = "moonlight"
+	force = 32					//One of the best support weapons. Does HE damage in it's stead.
+	damtype = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+	attack_verb_continuous = list("beats", "jabs")
+	attack_verb_simple = list("beat", "jab")
+	var/inuse
+
+/obj/item/ego_weapon/moonlight/attack_self(mob/user)
+	. = ..()
+	if(!CanUseEgo(user))
+		return
+
+	if(inuse)
+		return
+	inuse = TRUE
+	if(do_after(user, 30))	//3 seconds for a big heal.
+		playsound(src, 'sound/magic/staff_healing.ogg', 200, FALSE, 9)
+		for(var/mob/living/carbon/human/L in range(5, get_turf(user)))
+			L.adjustSanityLoss(10)
+	inuse = FALSE
+
+
+/obj/item/ego_weapon/heaven
```

Comment:
Why is THIS here?

Reply:
Just some bonus, I had the sprites, and wanted to make use of it. 
It doesn't affect anything, gameplay wise.